PocketKit/Sources/Sync/Slates/SlateService.swift

```diff
@@ -22,11 +22,15 @@ class APISlateService: SlateService {
     func fetchSlateLineup(_ identifier: String) async throws {
         let query = GetSlateLineupQuery(lineupID: identifier, maxRecommendations: 5)
 
-        guard let remote = try await apollo.fetch(query: query).data?.getSlateLineup else {
-            return
+        do {
+            guard let remote =  try await apollo.fetch(query: query).data?.getSlateLineup else { return }
+            try await handle(remote: remote)
+
+        } catch {
+            print("error \(error)")
+            throw error
```

Comment:
This is some debug code left over from our rubber ducky session. Let's try and figure out if this error is bubbling up correctly, and also determine how to best handle this error (revert to rethrowing, or capturing here. Either way, remove the print).

Reply:
Done.